src/remux/mp4-remuxer.js

```diff
@@ -270,17 +273,12 @@ class MP4Remuxer {
         firstDTS = nextAvcDts;
         inputSamples[0].dts = firstDTS;
         // offset PTS as well, ensure that PTS is smaller or equal than new DTS
-        firstPTS = Math.max(firstPTS - delta, nextAvcDts);
-        inputSamples[0].pts = firstPTS;
-        logger.log(`Video: PTS/DTS adjusted: ${toMsFromMpegTsClock(firstPTS, true)}/${toMsFromMpegTsClock(firstDTS, true)}, delta: ${toMsFromMpegTsClock(delta, true)} ms`);
+        minPTS = Math.max(minPTS - delta, nextAvcDts);
+        inputSamples[0].pts = minPTS;
```

Comment:
This is maybe risky since samples do not have to be in PTS order - they are in DTS order first.

Should we clamp all PTS values to a minimum of the new  `minPTS` here?

Or touch the first sample's PTS and update minPTS as necessary (seems wonky, but potentially less wonky than assuming first and min PTS are the same.)

```suggestion
        inputSamples[0].pts = Math.max(inputSamples[0].pts - delta, nextAvcDts);
        minPTS = Math.min(minPTS, inputSamples[0].pts);
```